docs/es/week12/12-2.md

```diff
@@ -0,0 +1,600 @@
+
+Para agregar contexto, podemos entrenar un modelo de lenguaje condicional. Luego, dado este modelo de lenguaje, que predice una palabra en cada paso de tiempo, reemplace cada salida del modelo con alguna otra característica.
+
+<!--- Pretraining - predict next word
+- Fine-tuning - change to a specific task. Examples:
+  - Predict whether noun or adjective
+  - Given some text comprising an Amazon review, predict the sentiment score for the review-->
+
+- Entrenamiento previo: predice la siguiente palabra
+- Ajuste fino: cambie a una tarea específica. Ejemplos:
+  - Predecir si es sustantivo o adjetivo
+  - Dado un texto que comprende una revisión de Amazon, predice la puntuación de sentimiento para la revisión
+
+<!--This approach is good because we can reuse the model. We pretrain one large model and can fine tune to other tasks.-->
+
+Este enfoque es bueno porque podemos reutilizar el modelo. Entrenamos previamente un modelo grande y podemos ajustarnos a otras tareas.
+
+<!--### ELMo-->
+
+### ELMo
+
+<!--GPT only considers leftward context, which means the model can't depend on any future words - this limits what the model can do quite a lot.-->
+
+GPT solo considera el contexto hacia la izquierda, lo que significa que el modelo no puede depender de ninguna palabra futura; esto limita mucho lo que el modelo puede hacer.
+Here the approach is to train _two_ language models
+
+<!--- One on the text left to right
+- One on the text right to left
+- Concatenate the output of the two models in order to get the word representation. Now can condition on both the rightward and leftward context.-->
+
+Aquí el enfoque es entrenar _dos_ modelos de lenguaje
+
+- Uno en el texto de izquierda a derecha
+- Uno en el texto de derecha a izquierda
+- Concatenar la salida de los dos modelos para obtener la representación de la palabra. Ahora puede condicionar tanto el contexto hacia la derecha como hacia la izquierda.
+
+<!--This is still a "shallow" combination, and we want some more complex interaction between the left and right context.-->
+
+Esta sigue siendo una combinación "superficial" y queremos una interacción más compleja entre el contexto izquierdo y derecho.
+
+<!--### BERT-->
+
+### BERT
+
+<!--BERT is similar to word2vec in the sense that we also have a fill-in-a-blank task. However, in word2vec we had linear projections, while in BERT there is a large transformer that is able to look at more context. To train, we mask 15% of the tokens and try to predict the blank.-->
+
+BERT es similar a word2vec en el sentido de que también tenemos una tarea de rellenar un espacio en blanco. Sin embargo, en word2vec teníamos proyecciones lineales, mientras que en BERT hay un gran transformador que es capaz de mirar más contexto. Para entrenar, enmascaramos el 15% de las fichas e intentamos predecir el espacio en blanco.
```

Comment:
Should we use "tokens" instead of "fichas" for consistency throughout the document?